### PR TITLE
overview page UI

### PR DESCRIPTION
**Pull request by dekguh**
loom: https://www.loom.com/share/0d52f9f5ab4742549f2338cf08e023c8

### CODE REVIEW

File: components/OverviewContainer/OverviewActivies.js [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+import Link from 'next/link'
+
+const OverviewActivies = () => {
+  return (
+    <div
+        style={{
+            padding: '0 15px',
+            marginTop: '8px'
+        }}
+    >
```
Comment (shonsirsha) [inline components/OverviewContainer/OverviewActivies.js]: Minta tolong jangan pakai inline styling mas untuk semua yg skrng pake inline styling sprt ini. Jadi, minta tolong pake bootstrap classes ya. Contoh, `px-2` dan `mt-1`. Ukuran pixel kalo ga pas dgn design gapapa, tapi ikuti saja yg terdekat pake bootstrap utility classes (bukan inline styling).

https://getbootstrap.com/docs/5.0/utilities/spacing/

Kalau mau dibikin pas dengan design, jangan inline styling juga tapi buat component baru dgn styled-components (bikin div ini styled pake styled-components).

File: components/OverviewContainer/OverviewActivies.js [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+        <TitleWithLink
+            title='Activities'
+            textLink='View in Block Explorer'
+            href='/activities'
+            style={{
+                marginBottom: '16px'
+            }}
```
Comment (shonsirsha) [inline components/OverviewContainer/OverviewActivies.js]: Please, dont use inline styling here too.

File: components/OverviewContainer/OverviewInventory.js [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+    <div style={{ marginTop: '24px' }}>
+        <div style={{ padding: '0 15px' }}>
+            <TitleWithLink
+                title='Inventory'
+                textLink='Open inventory'
+                href='/inventory'
+                style={{
+                    marginBottom: '8px'
+                }}
+            />
```
Comment (shonsirsha) [inline components/OverviewContainer/OverviewInventory.js]: Please, dont use inline styling here too.

File: components/OverviewContainer/OverviewInventory.js [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+ */
+const CardInventoryActive = ({ iconSrc, text }) => {
+    return(
+        <CardActiveWrap>
+            <img
+                src={iconSrc}
+                style={{
+                    marginTop: '4px'
+                }}
+            />
```
Comment (shonsirsha) [inline components/OverviewContainer/OverviewInventory.js]: Please, dont use inline styling here too, dan pake Image component dari NextJS mas (jgn pake img native html). `import Image from next/image` . 

Sudh banyak penggunaan Image (next/image) di repo ini, coba dicek ya mas kalau blm pernah pake.

File: components/OverviewContainer/OverviewInventory.js [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+
+const CardInventorySoon = () => {
+    return(
+        <CardSoonWrap>
+            <img
+                style={{
+                    marginTop: '3px'
+                }}
+                src='/images/question_mark.svg'
+            />
```
Comment (shonsirsha) [inline components/OverviewContainer/OverviewInventory.js]: Please, dont use inline styling here too, dan pake Image component dari NextJS mas (jgn pake img native html). import Image from next/image .

Sudh banyak penggunaan Image (next/image) di repo ini, coba dicek ya mas kalau blm pernah pake.

File: components/OverviewContainer/OverviewInventory.js [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+
+            <TextSoon>Artifacts</TextSoon>
+
+            <BlockComingSoon>
+                <span style={{ textAlign: 'center', display: 'block', width: '100%' }}>coming soon</span>
```
Comment (shonsirsha) [inline components/OverviewContainer/OverviewInventory.js]: Please, dont use inline styling here too. Use utility classes. E.g for text center use `text-center`.

Comment (shonsirsha) [reply components/OverviewContainer/OverviewInventory.js]: Kalau blm biasa pake bootstrap utility classes, bisa lihat di docs nya.

Mulai dari https://getbootstrap.com/docs/5.0/utilities/background/

File: components/OverviewContainer/OverviewWallet.js [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+ * Props:
+ * @param string addressEth
+ * @param number totalBtc
+ * @param string totalBtcUsd
+ * @param number totalEth1
+ * @param string totalEthUsd1
+ * @param number totalEth2
+ * @param string totalEthUsd2
+ * @returns JSX.Element
+ */
```
Comment (shonsirsha) [inline components/OverviewContainer/OverviewWallet.js:17]: Very nice!

File: components/OverviewContainer/OverviewWallet.js [hunk 8]
```diff
@@ -0,0 +1,233 @@
...
+                    <AddressTitle>Ethereum Address</AddressTitle>
+
+                    <CopyAddress>
+                        <AddressText>{addressEth}</AddressText>
+
+                        <button
+                            style={{
+                                backgroundColor: 'transparent',
+                                padding: 0,
+                                border: 'none'
```
Comment (shonsirsha) [inline components/OverviewContainer/OverviewWallet.js]: Please, dont use inline styling here too.

File: components/OverviewContainer/OverviewWallet.js [hunk 9]
```diff
@@ -0,0 +1,233 @@
...
+    <div
+        style={{
+            padding: '0 15px'
+        }}
+    >
+        <TitleWithLink
+            title='Wallet'
+            style={{
+                marginBottom: '8px'
+            }}
```
Comment (shonsirsha) [inline components/OverviewContainer/OverviewWallet.js]: Please, dont use inline styling here too,

File: components/Typography/TitleWithLink.js [hunk 10]
```diff
@@ -0,0 +1,62 @@
...
+ * Props:
+ * @param string title
+ * @param string textLink - optional (will hide when prop empty)
+ * @param string href - url of text link
+ * @param object style - optional (style for wrapper)
+ * @returns JSX.Element
+ */
+const TitleWithLink = ({ title, textLink, href = '#', style }) => {
+  return (
+    <div style={style}>
```
Comment (shonsirsha) [inline components/Typography/TitleWithLink.js]: Please, dont use inline styling here too.

File: components/OverviewContainer/OverviewWallet.js [hunk 11]
```diff
@@ -0,0 +1,233 @@
...
+                </AddressContent>
+            </AddressWrap>
+
+            <ListCrypto>
+                {/* item */}
+                <CryptoItem>
+                    <CryptoIcon>
+                        <img
+                            src='/images/Bitcoin.svg'
+                        />
```
Comment (shonsirsha) [inline components/OverviewContainer/OverviewWallet.js]: Tolong pakai next Image

File: components/OverviewContainer/OverviewWallet.js [hunk 12]
```diff
@@ -0,0 +1,233 @@
...
+                            style={{
+                                backgroundColor: 'transparent',
+                                padding: 0,
+                                border: 'none'
+                            }}
+                            onClick={() => navigator.clipboard.writeText(addressEth)}
+                        >
+                            <img
+                                src='/images/copy_all.svg'
+                            />
```
Comment (shonsirsha) [inline components/OverviewContainer/OverviewWallet.js]: tolong pakai next Image

File: components/OverviewContainer/OverviewWallet.js [hunk 13]
```diff
@@ -0,0 +1,233 @@
...
+                        <TextUsd>{totalBtcUsd}</TextUsd>
+                    </CryptoDetail>
+                </CryptoItem>
+
+                {/* item */}
+                <CryptoItem>
+                    <CryptoIcon>
+                        <img
+                            src='/images/Ethereum.svg'
+                        />
```
Comment (shonsirsha) [inline components/OverviewContainer/OverviewWallet.js]: tolong pakai next Image

File: components/OverviewContainer/OverviewWallet.js [hunk 14]
```diff
@@ -0,0 +1,233 @@
...
+                        <TextUsd>{totalEthUsd1}</TextUsd>
+                    </CryptoDetail>
+                </CryptoItem>
+
+                {/* item */}
+                <CryptoItem>
+                    <CryptoIcon>
+                        <img
+                            src='/images/Ethereum.svg'
+                        />
```
Comment (shonsirsha) [inline components/OverviewContainer/OverviewWallet.js]: tolong pakai next Image

File: components/OverviewContainer/OverviewInventory.js [hunk 15]
```diff
@@ -0,0 +1,162 @@
...
+import { mediaBreakpoint } from 'utils/breakpoints'
+
+const OverviewInventory = () => {
+  return (
+    <div style={{ marginTop: '24px' }}>
+        <div style={{ padding: '0 15px' }}>
+            <TitleWithLink
+                title='Inventory'
+                textLink='Open inventory'
+                href='/inventory'
```
Comment (shonsirsha) [inline components/OverviewContainer/OverviewInventory.js]: href ke /nbmons

File: components/OverviewContainer/OverviewActivies.js (hunk 1 above)
Comment (dekguh) [reply components/OverviewContainer/OverviewActivies.js]: oke siap mas, udh lama gk pakek bootstrap jadi agak lupa sama nama2 classnya

**Pull request merged**